app/controllers/feedback_controller.rb

```diff
@@ -1,35 +1,49 @@
 class FeedbackController < ApplicationController
+  include OrganizationScopable
+
   skip_before_action :authenticate_user!
-  skip_verify_authorized only: %i[new create]
-  layout :set_layout, only: %i[new create]
+  skip_verify_authorized only: %i[new create show]
+  layout :set_layout, only: %i[new]
+
+  def show
+  end
 
   def new
-    @contact = Contact.new
+    @feedback = Feedback.new
   end
 
   def create
-    @contact = Contact.new(contact_params)
-    if @contact.valid?
-      FeedbackMailer.with(contact_params).send_message.deliver_later
-      redirect_to root_path, notice: I18n.t("contacts.create.success")
+    @feedback = Feedback.new(feedback_params)
+
+    if @feedback.valid?
+      FeedbackMailer.with(feedback_params).send_message.deliver_later
+      path = case set_layout
```

Comment:
Can we move this to a private method e.g., `set_redirect_path` or the likes

Reply:
I moved it to a private method. I don't think we need a before_action.